src/Fixer/Phpdoc/PhpdocToCommentFixer.php

```diff
@@ -102,15 +119,23 @@ public function configure(array $configuration = null): void
             static fn (string $tag): string => strtolower($tag),
             $this->configuration['ignored_tags']
         );
+
+        $this->allowBeforeReturnStatement = (bool) $this->configuration['allow_before_return_statement'];
     }
 
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
+        // @TODO: In 4.0, make allow_before_return_statement option true by default
+
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('ignored_tags', 'List of ignored tags (matched case insensitively).'))
                 ->setAllowedTypes(['array'])
                 ->setDefault([])
                 ->getOption(),
+            (new FixerOptionBuilder('allow_before_return_statement', 'Whether to allow PhpDoc before return statement.'))
+                ->setAllowedTypes(['bool'])
+                ->setDefault(false)
```

Comment:
```suggestion
        return new FixerConfigurationResolver([
            (new FixerOptionBuilder('ignored_tags', 'List of ignored tags (matched case insensitively).'))
                ->setAllowedTypes(['array'])
                ->setDefault([])
                ->getOption(),
            (new FixerOptionBuilder('allow_before_return_statement', 'Whether to allow PhpDoc before return statement.'))
                ->setAllowedTypes(['bool'])
                ->setDefault(false) // @TODO 4.0: set to `true`
```

(just to mimic other cases like this)

Reply:
Done